src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
-}
+  for (const action of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
    const { type, extraData , keysToRemove } = action;
    
    switch (type) {
```
this destructuring was meant)